frontend/components/ui/TransactionSteps.tsx

```diff
@@ -1,14 +1,43 @@
 import { LoadingOutlined } from '@ant-design/icons';
-import { Steps, Typography } from 'antd';
+import { Steps as AntdSteps, Typography } from 'antd';
 import { FC, ReactNode } from 'react';
 import styled from 'styled-components';
 
+import { COLOR } from '@/constants/colors';
 import { UNICODE_SYMBOLS } from '@/constants/symbols';
 import { BridgingStepStatus } from '@/types/Bridge';
 import { Nullable } from '@/types/Util';
 
 const { Text } = Typography;
 
+const Steps = styled(AntdSteps)`
```

Comment:
is it copy-pasted from frontend/components/Bridge/BridgeInProgress/BridgingSteps.tsx? any chance we can reuse it maybe by moving to frontend/components/ui?

Reply:
ah right, they are very similar. my bad.